### PR TITLE
remove CreateBootstrap opting-in 

### LINKED ISSUE #10751: [Documentation] Remove the mentions of CreateBootstrap property from our docs
### Context

https://github.com/dotnet/msbuild/issues/10717#issuecomment-2391453471

The mentions of `CreateBootstrap` are confusing. The property has no effect - let's remove it from the docs.

cc: @jrdodds

### PR DESCRIPTION
**Pull request by JanProvaznik**
Fixes #10751

### Context
Bootstrap is on by default, opt-in with arg `/p:CreateBootstrap=true` is unnecessary

### Changes made
docs, CI: remove instances of `CreateBootstrap=true`

### CODE REVIEW

File: documentation/wiki/Building-Testing-and-Debugging-on-Full-Framework-MSBuild.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Please see [Contributing Code](https://github.com/dotnet/msbuild/blob/main/docum
 To build projects using the MSBuild binaries from the repository, you first need to do a build which produces
 a "bootstrap" directory. The "bootstrap" directory mimics a Visual Studio installation by acquiring additional
 dependencies (Roslyn compilers, NuGet, etc.) from packages or from your local machine (e.g. props/targets
-from Visual Studio). This will happen by default when running `.\build.cmd`. The bootstrap can be disabled by running `.\build.cmd /p:CreateBootstrap=false`.
```
Comment (JanKrivanek) [inline documentation/wiki/Building-Testing-and-Debugging-on-Full-Framework-MSBuild.md:66]: Do we want to remove this?
The statement seems to be true: 

https://github.com/dotnet/msbuild/blob/c4d51a11b84b4aabd9e5da1e3099f2c7c85024fe/eng/BootStrapMsBuild.targets#L20-L22

File: eng/cibuild_bootstrapped_msbuild.ps1 [hunk 2]
```diff
@@ -118,10 +118,10 @@ try {
   # - Turn off node reuse (so that bootstrapped MSBuild processes don't stay running and lock files)
   # - Create bootstrap environment as it's required when also running tests
   if ($onlyDocChanged) {
-    & $PSScriptRoot\Common\Build.ps1 -restore -build -ci /p:CreateBootstrap=false /nr:false @properties
```
Comment (JanKrivanek) [inline eng/cibuild_bootstrapped_msbuild.ps1:121]: Let's avoid bootstrap creation when we change only the docs

File: eng/cibuild_bootstrapped_msbuild.sh [hunk 3]
```diff
@@ -84,8 +84,8 @@ export DOTNET_HOST_PATH="$_InitializeDotNetCli/dotnet"
 # - Create bootstrap environment as it's required when also running tests
 if [ $onlyDocChanged = 0 ]
 then
-    . "$ScriptRoot/common/build.sh" --restore --build --test --ci --nodereuse false --configuration $configuration /p:CreateBootstrap=true $properties $extra_properties
+    . "$ScriptRoot/common/build.sh" --restore --build --test --ci --nodereuse false --configuration $configuration $properties $extra_properties
 
 else
-    . "$ScriptRoot/common/build.sh" --restore --build --ci --nodereuse false --configuration $configuration /p:CreateBootstrap=false $properties $extra_properties
```
Comment (JanKrivanek) [inline eng/cibuild_bootstrapped_msbuild.sh:90]: Same here - let's keep the opt out

**Review by JanKrivanek — CHANGES_REQUESTED** [review]
Thanks for the change!

I'm to blame for the wrong formulation of the bug.

`CreateBootstrap` property can only be used to opt-out from bootstrap creation. We still want to keep this behavior (and that mention in the doc)

Bootstrap is created by default - so explicit opt-in is rendundand and cinfusing - that's what we want to remove

**Review by JanKrivanek — APPROVED** [review]
Thank you!

**Review by YuliiaKovalova — APPROVED** [review]

**Pull request merged**